### ISSUE TITLE
Search Bar Autocomplete and Screen Reader(s)

### ISSUE THREAD

**Issue by mtribone** [projectblacklight/blacklight#1907]
When testing Blacklight 7 with the available online demo (http://demo.projectblacklight.org/) our users with the accessibility group experienced difficulty with the autocomplete and the search bar. 

1. Two input boxes near each other, one being read only and the other being edit, was a little confusing. If 2 input boxes are to be used then add additional descriptive or help text to aid users.
1. Autocomplete doesn't announce itself when typing into the input. Screen reader users are unaware that other suggestions based on their input are occurring at the same time.
1. When typing a term into the search box and hitting TAB to get to the next form element, the action of hitting TAB selects the first result from the autocomplete. This is confusing to the user because the search is performed with a completely different set of search terms. TAB should not be used to select items from autocomplete.

**Comment by barmintor** [comment]
Cf #1905

**Comment by cbeer** [comment]
#2576 replaced the autocomplete implementation, and hopefully/presumably github's is better.
